app/assets/stylesheets/application.scss

```diff
@@ -126,3 +126,12 @@ $navbar-brand-padding-y: 8px;
   height: 9.5em;
   width: 100%;
 }
+.nav-five img {
```

Comment:
@dixpac this was because the icons were mismatched. Should we try to convert over to all font-awesome icons in the nav so they can be uniform? or do we still want the customer SVG's for the other ones?

Reply:
Great question, I'm 👍 for uniform icons if possible, I will merge this as is for now. But, @abronkema about ☝️